source/OptimaJet.DWKit.StarterApplication/OptimaJet.DWKit.StarterApplication.csproj

```diff
@@ -19,12 +19,6 @@
     <MvcRazorCompileOnPublish>false</MvcRazorCompileOnPublish>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(RunConfiguration)' == 'Default' ">
-    <StartAction>Project</StartAction>
-    <LaunchBrowser>false</LaunchBrowser>
-    <ApplicationURL>http://localhost:5000</ApplicationURL>
-  </PropertyGroup>
-
```

Comment:
Don't know about this one one way or the other.  Just wanted to make sure this is intentional